src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -963,10 +963,12 @@ $(document).ready(() => {
     if (typeof richTextEditorBuilder !== 'undefined') {
         $.each(textFields, (i, textField) => {
             const id = $(textField).attr('id');
+            const isSessionClosed = $(textField).data('is-session-closed');
```

Comment:
It might be best to handle the case where the `data-is-session-closed` is undefined just in case. Can change this line to 
```JavaScript
const isSessionClosed = $(textField).data('is-session-closed') || false;
```

 I checked and realised that if the data is undefined then the text field becomes uneditable. I would think that making the `readonly` property `false` as the default case (to give the benefit of doubt to the session being open) might be desirable as we don't want any case in which the student is unable to submit feedback.

That being said, lets see what @damithc sir has to say about this suggestion before you implement it.